psi4/driver/procrouting/response/scf_response.py

```diff
@@ -256,164 +262,398 @@ def _print_output(complete_dict, output):
 
 
 def _print_tdscf_header(**options):
-    core.print_out('\n\n         ---------------------------------------------------------\n'
-                   '         {:^57}\n'.format('TDSCF excitation energies') +
-                   '         {:^57}\n'.format('by Andrew M. James and Daniel G. A. Smith') +
-                   '         ---------------------------------------------------------\n')
-
-    core.print_out("\n  ==> Requested Excitations <==\n\n")
-    state_info = options.pop('states')
-    for nstate, state_sym in state_info:
-        core.print_out("      {} states with {} symmetry\n".format(nstate, state_sym))
+    core.print_out("\n\n         ---------------------------------------------------------\n"
+                   f"         {'TDSCF excitation energies':^57}\n" +
```

Comment:
Just checking, are those ^57 characters supposed to be there?

Reply:
They make sure both lines are centered in the box.

Reply:
Ah, I see that now in my [go-to source](https://pyformat.info/#number_padding) for all things pyformat.  Thanks!